src/app/app.component.ts

```diff
@@ -7,5 +8,10 @@ import { Component } from '@angular/core';
 })
 
 export class AppComponent {
-  title = 'app';
+
+  eventsService: EventsService;
```

Comment:
Is this being used?

Reply:
I thought not but when I removed it everything blown up :D The idea is that this initializes the socket connection when the app starts, so that even if you don't add a song, the app is already listening to upcoming events. May be needed ahead when the log-in event is sent.